src/separator/io/Parser.java

```diff
@@ -319,7 +320,16 @@ public PersistentVector parseRow() throws IOException {
                 }
                 row.add(cell);
                 if (lastSeenSentinel != Sentinel.SEP) {
-                    return PersistentVector.create(row);
+                    PersistentVector vec = PersistentVector.create(row);
+                    int line = reader.getLineNumber();
+                    int column = reader.getColumnNumber();
+                    Object[] elements = new Object[4];
+                    elements[0] = Keyword.intern(null, "line");
+                    elements[1] = line;
+                    elements[2] = Keyword.intern(null, "column");
+                    elements[3] = column;
```

Comment:
Minor ⛳ since we don't need the `line` or `column` again:
```suggestion
                    Object[] elements = new Object[4];
                    elements[0] = Keyword.intern(null, "line");
                    elements[1] = reader.getLineNumber();
                    elements[2] = Keyword.intern(null, "column");
                    elements[3] = reader.getColumnNumber();
```